etc/zuul/openlab-main.yaml

```diff
@@ -69,6 +69,10 @@
               exclude-unprotected-branches: true
           - theopenlab/envoy:
               exclude-unprotected-branches: true
+          - containerd/containerd:
+              exclude-unprotected-branches: true
+          - envoyproxy/envoy:
+              exclude-unprotected-branches: true
```

Comment:
@Yikun 
I didn't found there is protected branch of `containerd` repo, and there are 3 branches of `envoy` are protected.
you can check with links:
https://github.com/containerd/containerd/branches
https://github.com/envoyproxy/envoy/branches

Reply:
And why we have to add these two project both under the theopenlab and the community? Or check one of them?

Reply:
1. the containerd/containerd is for periodic job. (don't need protected branches, just for add a project let zuul know this project)
2. the theopenlab/containerd is for trigger a pr job.

Reply:
> 1. the containerd/containerd is for periodic job. (don't need protected branches, just for add a project let zuul know this project)
you have configured `exclude-unprotected-branches: true` for `containerd/containerd` which has no protected branch, zuul will not load the project, the periodic job of this will never run. periodic job also need to consider protected branch.

Reply:
Why we need two types job for the project? the job is same or different? Or can we use two pipelines for the jobs? 'check' and 'periodic-*'.